pages/common/az-login.md

```diff
@@ -0,0 +1,29 @@
+# az login
+
+> Login to Azure.
+> Part of `az`.
+> More information: <https://docs.microsoft.com/cli/azure/reference-index#az_login>.
+
+- Login interactively:
+
+`az login`
+
+- Login with a username and password:
+
+`az login --username {{alias@somedomain.com}} --password {{secret}}`
+
+- Login with a service principal using a client secret:
+
+`az login --service-principal --username {{http://azure-cli-service-principal}} --passsword {{secret}} --tenant {{someone.onmicrosoft.com}}`
+
+- Login with a service principal using a client certificate:
+
+`az login --service-principal --username {{http://azure-cli-service-principal}} --password {{path/to/cert.pem}} --tenant {{someone.onmicrosoft.com}}`
+
+- Login using a VM's system assigned identity:
+
+`az login --identity`
+
+- Login using a VM's user assigned identity:
+
+`az login --identity --username {{/subscriptions/subscription_id/resourcegroups/my_rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/my_id}}`
```

Comment:
If this username has to start that way, I think it's more readable like this:
```suggestion
`az login --identity --username /subscriptions/subscription_id/resourcegroups/my_rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{{my_id}}`
```

Reply:
On second thoughts ... it does contain 3 moving parts .. the `subscription_id`, `resourcegroup` & `id` .. so either we put the whole thing as it is easily available both via the cli & portal

Reply:
I'm fine with the current solution. It creates way more readable output:
![image](https://user-images.githubusercontent.com/50295997/125759236-240e5740-7700-406a-8fad-35d95358e9cf.png)

Reply:
@mfrw I think was a bit unclear. With current solution I meant my suggestion above.

Reply:
ohh sure .. my apologies